src/app/pipes/alm-moment-time.pipe.ts

```diff
@@ -10,11 +10,18 @@ import * as moment from 'moment';
 
 @Pipe({ name: 'almMomentTime'})
 export class AlmMomentTime implements PipeTransform {
-  transform(val: string): any {    
+  transform(val: any): any {
```

Comment:
It should be changed to string. Date object should be able to build from string and string is more specific than any.